srcpkgs/dotnet-sdk/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'dotnet-sdk'
+pkgname=dotnet-sdk
+version=2.2.203
+revision=1
+archs="x86_64 armv7 aarch64"
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
+short_desc="Provides the .NET SDK components"
+maintainer="Anachron <gith@cron.world>"
+license="MIT"
+homepage="https://dotnet.microsoft.com"
+nopie=yes
+
+case "${XBPS_TARGET_MACHINE}" in
+	x86_64)
+	_arch="x64"
+	_path="647f8505-3bf0-48c5-ac0f-3839be6816d7/d0c2762ded5a1ded3c79b1e495e43b7c"
+	distfiles="https://download.visualstudio.microsoft.com/download/pr/${_path}/${pkgname}-${version}-linux-${_arch}.tar.gz"
+	checksum="d1e2368b1335a6a5f496b887950ad5da3d85783ec76f74a663214989817bb497"
+	;;
+	armv7)
+	_arch="arm"
+	_path="e5573b57-df74-4b5b-8cd8-06973b66c3ac/b9ee29318ad2d87fa05adfaf74a8271c"
+	distfiles="https://download.visualstudio.microsoft.com/download/pr/${_path}/${pkgname}-${version}-linux-${_arch}.tar.gz"
+	checksum="76488566b7c81f12a517274562bf5bbb77e003cd9ca568f90d9bfb2865de3242"
+	;;
+	armv8)
```

Comment:
```suggestion
	aarch64)
```

Reply:
Wow, how could I miss that. Will be fixed asap.